### PR TITLE
Update Domain Blacklist

### PR DESCRIPTION
**Pull request by Zyresus**
Add more domain extensions to help filter more off-site links in chat, usernames, and especially statuses.

### CODE REVIEW

File: server/chat.js [hunk 1]
```diff
@@ -215,7 +215,7 @@ Chat.namefilter = function (name, user, forStatus = false) {
 		if (name.includes('@') && name.includes('.')) return '';
 
 		// url
-		if (/[a-z0-9]\.(com|net|org)/.test(name)) name = name.replace(/\./g, '');
+		if (/[a-z0-9]\.(com|us|uk|co|gg|tk|ml|gq|ga|xxx|download|stream|net|org|)/.test(name)) name = name.replace(/\./g, '');
```
Comment (Zarel) [inline server/chat.js]:
```suggestion
		if (/[a-z0-9]\.(com|us|uk|co|gg|tk|ml|gq|ga|xxx|download|stream|net|org|)\b/.test(name)) name = name.replace(/\./g, '');
```

`\b` is word boundary, right?

**Comment by Zarel** [comment]
Thanks!

**Comment by urkerab** [comment]
Well that explains what happened to my alt...

**Comment by Zarel** [comment]
What's your alt?

**Comment by urkerab** [comment]
It's now `rompsimus` but it used to be more relevant...

**Comment by Zarel** [comment]
Was it previously `rom.psim.us`?

**Comment by urkerab** [comment]
Indeed, obviously...

**Pull request merged**